### PR TITLE
add new build variables

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Adds the new variables for the new website frontend build

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [ ] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

File: .github/workflows/deploy-frontends-to-production.yml [hunk 1]
```diff
@@ -159,7 +159,14 @@ jobs:
       - name: Build and Push Docker Image
         run: |
           cd website/
-          docker build --build-arg REACT_WEB_STATIC_HOST=${{ secrets.REACT_WEB_STATIC_HOST }} --target=deployment --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-website:${{ needs.image-tag.outputs.build_id }} -f docker/Dockerfile .
+          docker build \
+          --target=deployment \
+          --build-arg REACT_WEB_STATIC_HOST=${{ secrets.REACT_WEB_STATIC_HOST }} \
+          --build-arg REACT_NETMANAGER_BASE_URL=${{ secrets.REACT_NETMANAGER_BASE_URL_PROD }} \
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:165]: @ja-odur , this is for the production link as required.

**Review by 123MwanjeMike — COMMENTED** [review]
Please ensure that all the new variables are added to the repo secrets
`REACT_NETMANAGER_BASE_URL_PROD=https://platform.airqo.net/api/v1/`
`REACT_NETMANAGER_BASE_URL=https://staging-platform.airqo.net/api/v1/`
`REACT_APP_BASE_AIRQLOUDS_URL=`
`REACT_APP_BASE_NEWSLETTER_URL=`

File: .github/workflows/deploy-frontends-to-production.yml (hunk 1 above)
Comment (NoahNsimbe) [reply .github/workflows/deploy-frontends-to-production.yml:165]: @ja-odur  and @Baalmart please help @123MwanjeMike add the new variables to the repo secrets

Comment (Baalmart) [reply .github/workflows/deploy-frontends-to-production.yml:165]: I am not sure about this procedure for updating the secrets of this particular project. Perhaps @ja-odur could assist. cc @123MwanjeMike

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**